shopfloor_mobile/static/wms/index.html

```diff
@@ -46,10 +47,12 @@
     <noscript>
       <strong>We're sorry but wms doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
     </noscript>
-    <div id="app">  
+    <div id="app">
       <v-app>
         <v-content>
-          <v-container>Hello world</v-container>
+          <!-- <v-container> -->
```

Comment:
@TDu to be removed?

Reply:
Well that is the change I changed with the push-force